www/about/policies/terms-of-service.md

```diff
@@ -89,17 +89,17 @@ Collaborator if either party breaches the terms of any such agreement (e.g. if
 the Collaborator fails to provide the promised goods or services).
 
   1. Gratipay does not determine how payments are allocated and is not
-     responsible for how Projects internally distribute payments. The agreement
-between the Project's Owner and Collaborators regarding the allocation of
+     responsible for how Project Owners distribute payments. The agreement
```

Comment:
I would say "any" agreement, not "the" agreement, since we don't know if there is any such agreement.